query.c

```diff
@@ -343,12 +341,15 @@ static void query_menu(char *buf, size_t buflen, struct Query *results, int retb
 
     menu->data = QueryTable = mutt_mem_calloc(menu->max, sizeof(struct Entry));
 
+    int i;
```

Comment:
This `i` is used in 6 different `for` loops.
I'm confident that each one isn't used beyond the scope of the `for`.

Reply:
Unfortunately all those loops begin with `for (i = 0, queryp = results;`.

Since `queryp` is not an `int`, I cannot change it to `for (i = 0, queryp = results;`.

Reply:
ah, fair enough